stable_baselines3/common/logger.py

```diff
@@ -23,6 +23,28 @@
 DISABLED = 50
 
 
+class Video(object):
+    """
+    Video data class storing the video frames and the frame per seconds
+    """
+
+    def __init__(self, frames: th.Tensor, fps: Union[float, int]):
+        self.frames = frames
+        self.fps = fps
+
+
+class FormatUnsupportedError(NotImplementedError):
+    def __init__(self, unsupported_formats: Sequence[str], value: Any):
```

Comment:
Just to avoid calling the `str(value)` function on each calling site where this error is raised, but you're right - it is easier to read and also gives potential users of the error class the possibility to log customized value descriptions.